app/uk/gov/hmrc/helptosave/controllers/HelpToSaveController.scala

```diff
@@ -16,26 +16,30 @@
 
 package uk.gov.hmrc.helptosave.controllers
 
+import cats.instances.future._
 import cats.instances.int._
+import cats.instances.string._
 import cats.syntax.eq._
 import com.google.inject.Inject
 import play.api.libs.json.{JsError, JsSuccess}
 import play.api.mvc.{Action, AnyContent, Result}
 import uk.gov.hmrc.auth.core.AuthConnector
 import uk.gov.hmrc.helptosave.audit.HTSAuditor
 import uk.gov.hmrc.helptosave.config.AppConfig
-import uk.gov.hmrc.helptosave.models.register.CreateAccountRequest
 import uk.gov.hmrc.helptosave.connectors.HelpToSaveProxyConnector
+import uk.gov.hmrc.helptosave.models.register.CreateAccountRequest
 import uk.gov.hmrc.helptosave.models.{AccountCreated, ErrorResponse, NSIPayload}
-import uk.gov.hmrc.helptosave.repo.EnrolmentStore
+import uk.gov.hmrc.helptosave.repo.{EmailStore, EnrolmentStore}
```

Comment:
could we add a unit test for delete email in the `MongoEmailStore` unit test please? I put the code in before but didn't write a unit test for it since at the time the plan was to throw it away soon after.